ocaml/xapi/repository.ml

```diff
@@ -373,7 +373,9 @@ let get_repository_handler (req : Http.Request.t) s _ =
   let open Xapi_stdext_std.Xstringext in
   debug "Repository.get_repository_handler URL %s" req.Request.uri ;
   req.Request.close <- true ;
-  if is_local_pool_repo_enabled () then (
+  if Fileserver.access_forbidden req s then
```

Comment:
> This commit restricts the service port of the local YUM repository on
the pool master to TLS interface only.

FYI this change will not block plain http requests on the local unix socket, but I assume that is allowed